tools/release-docs

```diff
@@ -1,115 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
In `RELEASE.md` we mention:

> Publish release documentation using `./tools/release-docs`

Could you update `RELEASE.md`? I guess we don't have to do anything because it being published automatically by CI, whenever we tag the release.

Reply:
Updated in [664cfa7](https://github.com/grafana/mimir/pull/1820/commits/664cfa7a09477bc1f680914e6599444f760ac860)